src/handler_error.rs

```diff
@@ -349,6 +353,18 @@ pub fn internal_json_encoding_failed_error(
     }
 }
 
+pub fn unexpected_sdk_error() -> Error {
+    Error {
+        code:        9100,
+        message:     "an unexpecred error related to the rust-SDK has occured, this should not be \
+                      possible"
+            .to_string(),
+        description: None,
+        retriable:   true,
+        details:     None,
+    }
+}
```

Comment:
It's unclear why this is a function seeing that it has no arguments and doesn't do any effects.